### ISSUE TITLE
Why pickscore reward has train.beta as 0?

### ISSUE THREAD

**Issue by tingxueronghua** [yifan123/flow_grpo#23]
Thanks for your great work! I am trying to reproduce the flow grpo on other models and have some questions to inquire.

I think this hyperparameter, train.beta, is used for weighting the KL loss. If setting it to 0, the KL loss does not take effect. Why does this work?

By the way, I have tested the flow grpo on other models and found that in some cases, the reward scores even decrease. Have you observed similar cases? Is this caused by the sensitivity to hyperparameters?

**Comment by yifan123** [comment]
I'm also experimenting with other models myself, such as video models. Initially, after migrating, I noticed the reward kept dropping. After a few days of debugging, I found it was due to a minor bug introduced during the migration process.  

When first trying a new model, I recommend setting `train.beta` to 0 to observe the upper limit of optimization. `train.beta` is used to prevent reward hacking, and setting it to 0 allows the reward to increase faster.  

My suggestion is that after migrating to a new model, start with the simplest reward, such as `aesthetic`. You should see a significant improvement in reward within 15 epochs—otherwise, there might be a bug in the implementation.

**Comment by yifan123** [comment]
This is normal because the `generate_image` is the image during training, which is the result of SDE sampling with 10 steps. An important conclusion in our paper is that this can significantly accelerate training, while using the original 40-step ODE to generate images (i.e., `eval_generate_image`) ensures normal quality and higher rewards. Additionally, isn't your training reward increasing? In my experience, the Pickscore tends to rise quite easily.

**Comment by sunhaha123** [comment]
> Thanks for your reply! It helps a lot. I have another question, is it normal to see images generated like this?
> 
> ![Image](https://github.com/user-attachments/assets/aa18b02f-8926-402a-8f0f-e8dd00c00cbb)
> 
> ![Image](https://github.com/user-attachments/assets/51895238-5bb3-457e-a125-a924459dfe6d)
> 
> I trained the model with reward models as MPS and Pickscore, separately. The reward_avg seems to be normal (at least not decrease), but the "generate_image" looks strange as shown above. The "eval_generate_image" looks much more normal.

@yifan123 
Using these images, scoring them with a model fine-tuned based on Vision LLM seems to be a bit problematic. It seems that Vision LLM considers them to be in the style of oil paintings, so it gave them quite good scores. 

Another thought is, if I want to use these images to improve certain physical accuracy of person, such as  hands, would it be feasible to use images with noise for judging?
